test-project-windows/Packages/manifest.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "com.unity.ide.visualstudio": "2.0.12"
+  }
```

Comment:
I'm not an expert at Unity, could you please explain why this would be needed as a dependency?

Reply:
Gonna be honest, I'm not too sure either. I think that got auto added when I opened the project to configure it. We might be able to just remove it.

Reply:
It's used to enable some integrations that Visual Studio has with Unity (like connecting to the Unity process, so you are able to setup breakpoints for debugging). I'd recommend either a) removing it or b) adding for all other IDEs like "com.unity.ide.rider" and "com.unity.ide.vscode". Probably a) is better.